src/core/Slider/Thumb.tsx

```diff
@@ -0,0 +1,98 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { Tooltip, TooltipProps } from '../Tooltip';
+
+/**
+ * Thumb is a local component used to show and modify the values maintained by the Slider.
+ * Only one Thumb can be active at a time. A Thumb is made active when the user selects
+ * it with pointer. Whenever a Thumb is active, focused, or hovered its tooltip is shown. */
+export const Thumb = ({
+  value,
+  index,
+  minVal,
+  maxVal,
+  step,
+  sliderMin,
+  sliderMax,
+  isActive,
+  onThumbActivated,
+  onThumbValueChanged,
+  tooltipContent,
+  tooltipProps,
+}: {
+  value: number;
+  index: number;
+  minVal: number;
+  maxVal: number;
+  step: number;
+  sliderMin: number;
+  sliderMax: number;
+  isActive: boolean;
+  onThumbActivated: (index: number) => void;
+  onThumbValueChanged: (index: number, value: number) => void;
+  tooltipContent: React.ReactNode;
+  tooltipProps?: Partial<Omit<TooltipProps, 'content' | 'children'>>;
+}) => {
+  const thumbRef = React.useRef<HTMLDivElement>(null);
+  const handleOnKeyDown = React.useCallback(
+    (event: React.KeyboardEvent<HTMLDivElement>) => {
+      if (event.key === 'ArrowLeft') {
+        const newValue = Math.max(value - step, minVal);
+        onThumbValueChanged(index, newValue);
+      } else if (event.key === 'ArrowRight') {
+        const newValue = Math.min(value + step, maxVal);
+        onThumbValueChanged(index, newValue);
+      } else if (event.key === 'Home') {
+        onThumbValueChanged(index, minVal);
+      } else if (event.key === 'End') {
+        onThumbValueChanged(index, maxVal);
+      }
+    },
+    [value, index, step, minVal, onThumbValueChanged, maxVal],
+  );
+
+  const handlePointerDownOnThumb = React.useCallback(
+    (event: React.PointerEvent) => {
+      thumbRef.current?.focus();
+      event.preventDefault();
+      event.stopPropagation();
+      onThumbActivated(index);
```

Comment:
After the latest css changes, we also need to apply `.iui-grabbing` on the slider container when thumb is active. [`eacd7e4` (#172)](https://github.com/iTwin/iTwinUI/pull/172/commits/eacd7e46e7c9d7f84b0d46c2616ab7cfb8d659a8)

Reply:
css is available in 0.24.0

Reply:
The class should be applied on the container, not the thumb.

Reply:
What, container? The parent of the slider? What is this new class doing?

Reply:
Yeah, check the commit I linked above. The new class fixes the cursor rapidly changing between 'grabbing' and 'pointer' when moving the slider (by overriding the container's cursor when the thumb is active).

This is how it currently behaves:
![cursor](https://user-images.githubusercontent.com/9084735/125516024-d68ece34-055a-4017-90fe-fd44d4557bc7.gif)

Reply:
Moved use of .iui-grabbing to the slider container.